server/src/utility/insert.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+const project = require("../models/Project");
+const domain = require("../models/Domain");
+
+const genCheckArr = (obj, key, elem) => {
+    if (Array.isArray(obj[key])) {
+        if (!obj[key].includes(elem)) {
+            obj[key].push(elem);
+        }
+    } else {
+        obj[key] = [elem];
+    }
+}
+
+const insertDomains = async () => {
+    await mongoose.connect(process.env.MONGO_URI);
+    let projects = await project.find({})
+    for (let i = 0; i < projects.length; i++) {
+        projects[i].domains = [projects[i].domains]
```

Comment:
```javascript
if(!Array.isArray(projects[i].domains)){
      projects[i].domains = [projects[i].domains]
}
```

This check should be made otherwise the every time this function is run it will create a higher dimension array.

Reply:
Good catch, maybe we use the `genCheckArr` here as well

Reply:
Updated the same.